admin/src/test/scala/za/co/absa/spline/admin/AdminCLISpec.scala

```diff
@@ -93,6 +96,17 @@ class AdminCLISpec
       } should include("--help")
     }
 
+    it should "when calling with option -k, create a non-validating SSLContext" in {
```

Comment:
```suggestion
    it should "create a non-validating SSLContext when called with option -k, " in {
```
Not important, but just bit nicer to read.

Reply:
Changed to "called", but I would leave the phase arrangement as is because it's consistent with the rest of "should" statements in the class. And it also visually groups related statements in test logs and in the code.

![image](https://user-images.githubusercontent.com/795479/123687711-8b8a9d80-d851-11eb-813b-371181a8d609.png)

Reply:
Since flatspec is going for "english sentence like" names, I would personally change all of them to feel like natural language, but I can live with the current version.